java.md

```diff
@@ -10,6 +10,8 @@ performance benefit until they switched to Java 11.
 This page includes specific details about building and tuning Java application on Graviton.
 
 ### Java versions
+If you are still using JDK8, please stronly consider upgrading. There are significant performance improvements that you can benefit from, such as better JIT code generation, optimized GC at high thread counts, Arm-tuned Java monitors and locks, an improved SpinPause logic (with even more options in JDK17, like OnSpinWaitInstCount and OnSpinWaitInst), and using Arm intrinsics.
```

Comment:
s/stronly/strongly/

Reply:
The formatting here (line length) is unlike the surrounding text. Not that it matters for the rendered result, of course.

Reply:
If you have them, links for some, or all, of those optimizations would help.

Reply:
I fixed the formatting and spelling, but I didn't add any links.

Reply:
We should rewrite this section with our up-to-date recommendations (>=Corretto17) and put the disclaimer about Corretto8 towards the end.